From 3989ba456713633b0bb1c3bbc5b5e55d9957b1ad Mon Sep 17 00:00:00 2001
From: "/setup.sh" <support@vision-components.com>
Date: Wed, 27 Nov 2024 16:37:36 +0100
Subject: [PATCH] Added VC MIPI Driver sources to Makefile.

---
 kernel/nvidia/drivers/media/i2c/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/nvidia/drivers/media/i2c/Makefile b/kernel/nvidia/drivers/media/i2c/Makefile
index c6d34e8..840aa0a 100644
--- a/kernel/nvidia/drivers/media/i2c/Makefile
+++ b/kernel/nvidia/drivers/media/i2c/Makefile
@@ -12,7 +12,7 @@ obj-$(CONFIG_VIDEO_IMX274) += imx274.o
 obj-$(CONFIG_VIDEO_IMX318) += imx318.o
 obj-$(CONFIG_VIDEO_LC898212) += lc898212.o
 obj-$(CONFIG_VIDEO_OV5693) += ov5693.o
-obj-$(CONFIG_VIDEO_OV9281) += ov9281.o
+obj-$(CONFIG_VIDEO_OV9281) += vc_mipi_camera.o vc_mipi_core.o vc_mipi_modules.o
 obj-$(CONFIG_VIDEO_OV10823) += ov10823.o
 obj-$(CONFIG_VIDEO_OV23850) += ov23850.o
 obj-$(CONFIG_I2C_IOEXPANDER_PCA9570) += pca9570.o
-- 
2.34.1

