From 5e0e8dc110e7f45013ea866bd03d95447ae3a9d0 Mon Sep 17 00:00:00 2001
From: Peter Martienssen <peter.martienssen@liquify-consulting.de>
Date: Tue, 28 Sep 2021 15:05:19 +0200
Subject: [PATCH 4/4] Added VC MIPI Driver sources to Makefile.

---
 kernel/nvidia/drivers/media/i2c/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/nvidia/drivers/media/i2c/Makefile b/kernel/nvidia/drivers/media/i2c/Makefile
index c6d34e807..840aa0ad5 100644
--- a/kernel/nvidia/drivers/media/i2c/Makefile
+++ b/kernel/nvidia/drivers/media/i2c/Makefile
@@ -12,7 +12,7 @@ obj-$(CONFIG_VIDEO_IMX274) += imx274.o
 obj-$(CONFIG_VIDEO_IMX318) += imx318.o
 obj-$(CONFIG_VIDEO_LC898212) += lc898212.o
 obj-$(CONFIG_VIDEO_OV5693) += ov5693.o
-obj-$(CONFIG_VIDEO_OV9281) += ov9281.o
+obj-$(CONFIG_VIDEO_OV9281) += vc_mipi_camera.o vc_mipi_core.o vc_mipi_modules.o
 obj-$(CONFIG_VIDEO_OV10823) += ov10823.o
 obj-$(CONFIG_VIDEO_OV23850) += ov23850.o
 obj-$(CONFIG_I2C_IOEXPANDER_PCA9570) += pca9570.o
-- 
2.25.1

