From 3c1cd44036653fecf433bc2a8be573f400004754 Mon Sep 17 00:00:00 2001
From: Peter Martienssen <peter.martienssen@liquify-consulting.de>
Date: Fri, 13 Aug 2021 17:05:29 +0200
Subject: [PATCH 6/9] Added VC MIPI Driver sources to Makefile.

Signed-off-by: Peter Martienssen <peter.martienssen@liquify-consulting.de>
---
 kernel/nvidia/drivers/media/i2c/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/nvidia/drivers/media/i2c/Makefile b/kernel/nvidia/drivers/media/i2c/Makefile
index c6d34e807..579d3f633 100644
--- a/kernel/nvidia/drivers/media/i2c/Makefile
+++ b/kernel/nvidia/drivers/media/i2c/Makefile
@@ -13,6 +13,7 @@ obj-$(CONFIG_VIDEO_IMX318) += imx318.o
 obj-$(CONFIG_VIDEO_LC898212) += lc898212.o
 obj-$(CONFIG_VIDEO_OV5693) += ov5693.o
 obj-$(CONFIG_VIDEO_OV9281) += ov9281.o
+obj-$(CONFIG_VIDEO_OV9281) += vc_mipi_camera.o vc_mipi_core.o vc_mipi_modules.o
 obj-$(CONFIG_VIDEO_OV10823) += ov10823.o
 obj-$(CONFIG_VIDEO_OV23850) += ov23850.o
 obj-$(CONFIG_I2C_IOEXPANDER_PCA9570) += pca9570.o
-- 
2.25.1

