From 7686c8063e8e63145d65a8961dfdd9c8edd62dc1 Mon Sep 17 00:00:00 2001
From: Peter Martienssen <peter.martienssen@liquify-consulting.de>
Date: Mon, 30 Aug 2021 19:24:27 +0200
Subject: [PATCH 8/9] Added all VC MIPI Driver files to .gitignore.

---
 .gitignore | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index b88916b64..d00421338 100644
--- a/.gitignore
+++ b/.gitignore
@@ -2,4 +2,6 @@ build/
 modules/
 *.*bz2*
 nv*.sh
-public_sources_sha.txt
\ No newline at end of file
+public_sources_sha.txt
+*vc_mipi*
+*vc-mipi*
-- 
2.25.1

