From 3a75f148bddd0056bb66f5eb22d48c30ffcd6fd0 Mon Sep 17 00:00:00 2001
From: "/setup.sh" <support@vision-components.com>
Date: Mon, 16 Dec 2024 10:19:32 +0100
Subject: [PATCH] Added VC MIPI Driver sources to nvidia-oot Makefile.

---
 nvidia-oot/drivers/media/i2c/Makefile | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/nvidia-oot/drivers/media/i2c/Makefile b/nvidia-oot/drivers/media/i2c/Makefile
index 866fc75..04c93f8 100644
--- a/nvidia-oot/drivers/media/i2c/Makefile
+++ b/nvidia-oot/drivers/media/i2c/Makefile
@@ -20,4 +20,11 @@ obj-m += nv_ar0234.o
 obj-m += pca9570.o
 obj-m += nv_hawk_owl.o
 obj-m += virtual_i2c_mux.o
+
+obj-m += vc_mipi_core.o
+vc_mipi_core-objs := vc_mipi/vc_mipi_core.o vc_mipi/vc_mipi_modules.o
+
+obj-m += vc_mipi_camera.o
+vc_mipi_camera-objs := vc_mipi/vc_mipi_camera.o
+
 endif
-- 
2.34.1

