From a8d7ffaeb93107f7d96940061d3e3b50d23e0ef6 Mon Sep 17 00:00:00 2001
From: Peter Martienssen <peter.martienssen@liquify-consulting.de>
Date: Mon, 27 Sep 2021 16:21:52 +0200
Subject: [PATCH] Added .gitignore

---
 .gitignore | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 000000000..fdb45f3e3
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,8 @@
+.vscode*
+build/
+modules/
+*.*bz2*
+nv*.sh
+public_sources_sha.txt
+*vc_mipi*
+*vc-mipi*
\ No newline at end of file
-- 
2.25.1

