From 86605db4e807779ed0dab8f41b4863da0829ddac Mon Sep 17 00:00:00 2001
From: pmliquify <peter.martienssen@liquify-consulting.de>
Date: Wed, 14 Jul 2021 16:49:39 +0200
Subject: [PATCH 1/1] Added .gitignore

---
 .gitignore | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 000000000..b88916b64
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,5 @@
+build/
+modules/
+*.*bz2*
+nv*.sh
+public_sources_sha.txt
\ No newline at end of file
-- 
2.25.1

